### PR TITLE
fix missing format string in #9171 - dbs version

### PR DESCRIPTION
**Pull request by amaltaro**
dbs branch of #9177 (first commit only)

@yuyiguo can you please test this bug fix and let me know if we can merge and cut a new dbs tag? Thanks

### CODE REVIEW

File: src/python/WMCore/WebTools/Root.py [hunk 1]
```diff
@@ -92,10 +91,10 @@ def access(self):
             # request.rfile.rfile.bytes_read is a custom CMS web
             #  cherrypy patch not always available, hence the test
             rbytes = (getattr(request.rfile, 'rfile', None)
-                 and getattr(request.rfile.rfile, "bytes_read", None)
-                 and request.rfile.rfile.bytes_read) or "-"
+                      and getattr(request.rfile.rfile, "bytes_read", None)
```
Comment (yuyiguo) [inline src/python/WMCore/WebTools/Root.py]: 8 spaces here instead of 4 ?

**Comment by yuyiguo** [comment]
@amaltaro 
Allan, The code here are tested. I hand picked the code to my test area so I missed the missed of "s" format in #9171.  The #9171 is combined of new stuff with #9169 that has the correct string format. I apply #9169 first so my test version was correct. Thank you for finding the missing "s" error in #9171 .

This pr looked good to me, except for a small thing on the code format that I have commented on the code. It should not be any functional issue. Please commit it and give the 1.1.16_dbs branch a new tag.
Thanks,
yuyi

File: src/python/WMCore/WebTools/Root.py (hunk 1 above)
Comment (amaltaro) [reply src/python/WMCore/WebTools/Root.py]: that's what PyCharm decided to put in place, I guess to keep it aligned with the whole statement.

**Comment by amaltaro** [comment]
@yuyiguo tag `1.1.16.dbs3` should be available. Thanks

**Comment by yuyiguo** [comment]
Thanks Alan. 
I hope this is the final tag needed for May release. Making a new PR to Lina now.

**Pull request merged**